chickenlittle.go

```diff
@@ -7,20 +7,20 @@ import (
 	"net/http"
 	"path/filepath"
 
+	"github.com/chrissnell/chickenlittle/ne"
 	"github.com/gorilla/mux"
 	"gopkg.in/yaml.v2"
 )
 
 var (
-	cfgFile  *string
-	c        ChickenLittle
-	NIP      NotificationsInProgress
-	planChan = make(chan *NotificationRequest)
+	cfgFile *string
+	c       ChickenLittle
 )
 
 type ChickenLittle struct {
```

Comment:
I recently saw another project that used an object called "Service" to hold all of the shared data like we're doing here with the ChickenLittle struct.   Once we get this branch merged, I want to go in and rename this struct to Service, because that makes more sense, e.g. Service.Config, Service.DB, etc.